x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/shared/SharedBlobCacheService.java

```diff
@@ -1606,6 +1631,24 @@ public int forceEvict(Predicate<KeyType> cacheKeyPredicate) {
             return evictedCount;
         }
 
+        @Override
+        public void forceEvictAsync(Predicate<KeyType> cacheKeyPredicate) {
+            evictionsRunner.enqueueTask(new ActionListener<>() {
+                @Override
+                public void onResponse(Releasable releasable) {
+                    forceEvict(cacheKeyPredicate);
```

Comment:
It's important to close the releasable:
```suggestion
  try (releasable) {
    forceEvict(cacheKeyPredicate);
  }
```